ios-trivia-game/ViewControllers/GameOptionsViewController.swift

```diff
@@ -10,17 +10,64 @@ import UIKit
 
 class GameOptionsViewController: UIViewController {
 
+    @IBOutlet weak var categoryPicker: UIPickerView!
+    @IBOutlet weak var numOfQuestionsPicker: UIPickerView!
+    
+    var categoryPickerData: [String] = [String]()
+    var numOfQuestionsPickerData: [String] = [String]()
+    
+    var numOfQuestions: Int?
+    
+    let pickerTagForCategory = 1;
```

Comment:
Yeah I wasn't sure about iOS convention, but agree with caps! I'll make that change!